packages/form-js-editor/test/spec/features/modeling/Modeling.spec.js

```diff
@@ -884,9 +884,8 @@ describe('features/modeling', function() {
 
     describe('group', function() {
 
-      const sourceIndex = 1;
-
       let formFieldIds,
+          sourceIndex,
```

Comment:
It's related to the test form, which changes every time there's a new component. I added the html component at the top which broke things. Just made the test robust to schema changes. Since changing this shouldn't have made it fail ideally.